code/game/objects/items/stunbaton.dm

```diff
@@ -173,9 +173,24 @@
 		var/mob/living/carbon/human/H = L
 		H.forcesay(GLOB.hit_appends)
 
-
 	return 1
 
+/// After a target is hit, we do a chunk of stamina damage, and apply a jitter and stutter.
+/// After a period of time, we then check to see if the target needs to have the stun applied.
+/obj/item/melee/baton/proc/stun_effect(mob/living/target)
+	target.Jitter(20)
+	target.apply_effect(EFFECT_STUTTER, stunforce)
+	target.adjustStaminaLoss(65)
+
+	SEND_SIGNAL(target, COMSIG_LIVING_MINOR_SHOCK)
+	addtimer(CALLBACK(src, .proc/apply_stun_effect_end, target), 25)
+
+/// After the initial stun period, we check to see if the target needs to have the stun applied.
+/obj/item/melee/baton/proc/apply_stun_effect_end(mob/living/target)
+	if(target.reagents.has_reagent(resistance_chem, 5, TRUE))
+		return

```

Comment:
Have you considered removing some of the resistance chem if this triggers, to allow brute forcing it? Maybe 3 of it.

Reply:
Yeah I like that

Reply:
pr changed since

Reply:
what was the change? I don't see it.

Reply:
It’s no longer epi chem, and the logic behind what happened when you “resist” the stun is different. Before, you completely ignored any drop at all, and removing the chem from your body in this case made sense. Now we have a new chem that let’s me tweak things a little more closely to what I want on that side, and the stun itself is still a stun, just shorter.